scripts/tests/scripts_regression_tests.py

```diff
@@ -585,7 +585,12 @@ def test_f_createnewcase_with_user_compset(self):
 
         cls._testdirs.append(testdir)
 
-        pesfile = os.path.join("..","src","drivers","mct","cime_config","config_pes.xml")
+        if CIME.utils.get_model() == "cesm":
```

Comment:
This test should be changed to use the nuopc driver. I have opened issue https://github.com/ESMCI/cime/issues/3920